android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -106,41 +106,79 @@ public void onBindViewHolder(RecyclerViewHolder holder, int position) {
+        if(feedItem.getId() != null) {
+
+            if (!TextUtils.isEmpty(feedItem.getMessage())) {
+                holder.statusMsg.setText(feedItem.getMessage());
+                holder.statusMsg.setVisibility(View.VISIBLE);
+            } else {
+                // status is empty, remove from view
+                holder.statusMsg.setVisibility(View.GONE);
+            }
+
+            // Checking for null feed url
+            if (feedItem.getLink() != null) {
+                holder.url.setText(Html.fromHtml("<a href=\"" + feedItem.getLink() + "\">"
+                        + feedItem.getLink() + "</a> "));
+
+                // Making url clickable
+                holder.url.setMovementMethod(LinkMovementMethod.getInstance());
+                holder.url.setVisibility(View.VISIBLE);
+            } else {
+                // url is null, remove from the view
+                holder.url.setVisibility(View.GONE);
+            }
+
+            String feedImageUri = Utils.parseImageUri(feedItem.getFullPicture());
+            Drawable placeholder = VectorDrawableCompat.create(context.getResources(),
+                    R.drawable.ic_placeholder_24dp, null);
+
+            if (feedImageUri != null) {
+                holder.feedImageView.setVisibility(View.VISIBLE);
+                Picasso.with(holder.feedImageView.getContext())
+                        .load(Uri.parse(feedImageUri))
+                        .placeholder(placeholder)
+                        .into(holder.feedImageView);
+            } else {
+                holder.feedImageView.setVisibility(View.GONE);
+            }
         } else {
-            // status is empty, remove from view
-            holder.statusMsg.setVisibility(View.GONE);
-        }
-
-        // Checking for null feed url
-        if (feedItem.getLink() != null) {
-            holder.url.setText(Html.fromHtml("<a href=\"" + feedItem.getLink() + "\">"
-                    + feedItem.getLink() + "</a> "));
-
-            // Making url clickable
-            holder.url.setMovementMethod(LinkMovementMethod.getInstance());
-            holder.url.setVisibility(View.VISIBLE);
-        } else {
-            // url is null, remove from the view
-            holder.url.setVisibility(View.GONE);
-        }
-
-        String feedImageUri = Utils.parseImageUri(feedItem.getFullPicture());
-        Drawable placeholder = VectorDrawableCompat.create(context.getResources(),
-                R.drawable.ic_placeholder_24dp, null);
 
-        if(feedImageUri != null) {
-            holder.feedImageView.setVisibility(View.VISIBLE);
-            Picasso.with(holder.feedImageView.getContext())
-                    .load(Uri.parse(feedImageUri))
-                    .placeholder(placeholder)
-                    .into(holder.feedImageView);
-        } else {
-            holder.feedImageView.setVisibility(View.GONE);
+            if (!TextUtils.isEmpty(feedItem.getText())) {
```

Comment:
Oh sorry . Got confused between text and message. Yes, getText will not be null if id is null since id corresponds to fb feed while text corresponds to loklak feed.